composer.json

```diff
@@ -45,5 +39,11 @@
         "psr-4": {
             "Heise\\Tests\\Shariff\\": "tests"
         }
-    }
+    },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/heiseonline/shariff-backend-php.git"
```

Comment:
I think this is not really needed. This is only needed in the composer.json of the project requiring this package.